src/docs/Glossary.adoc

```diff
@@ -33,37 +33,57 @@ Scheduler Web Interface:: ProActive component that provides a web interface to t
 [[_glossary_workflow_studio]]
 Workflow Studio:: ProActive component that provides a web interface for designing <<_glossary_workflow,*Workflows*>>.
 
+[[_glossary_automation_dashboard]]
+Automation Dashboard:: Centralized interface for the following web portals: <<_glossary_workflow_automation_portal,*Workflow Automation*>>, <<_glossary_catalog_portal,*Catalog*>>, <<_glossary_analytics_portal,*Analytics*>>, <<_glossary_job_planner_portal,*Job Planner*>>, <<_glossary_cloud_automation_portal,*Cloud Automation*>>, <<_glossary_cloud_watch_portal,*Cloud Watch*>> and <<_glossary_notification_portal,*Notifications*>>.
+
+[[_glossary_analytics]]
+Analytics::  ProActive service responsible to gather monitoring information for <<_glossary_job,*Jobs*>> and <<_glossary_proactive_node,*ProActive Nodes*>>
+
 [[_glossary_job_analytics]]
```

Comment:
Why we user **service** for some modules and **component** for others

Reply:
I thought I will receive such comment. I could switch to service eventually, but in some cases it does not apply.